pkgs/development/python-modules/strategies/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi
+, multipledispatch, toolz
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "strategies";
+  version = "0.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "02i4ydrs9k61p8iv2vl2akks8p9gc88rw8031wlwb1zqsyjmb328";
+  };
+
+  propagatedBuildInputs = [ multipledispatch toolz ];
+  checkInputs = [ pytest ];
+
+  meta = {
```

Comment:
Missing ``with stdenv.lib;``.

Reply:
Think that is fine here

Reply:
Sympy doesn't have it too: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/sympy/default.nix#L37

Reply:
Does seem unusual to have lib as an argument at the top just for meta though

Reply:
It's just a syntactical difference and a lot of packages skip the `with stdenv.lib` part. Resolving this for now as it's just a question of style.